x-pack/plugins/monitoring_collection/server/lib/get_kibana_stats.ts

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { i18n } from '@kbn/i18n';
+import { ServiceStatus, ServiceStatusLevels } from '../../../../../src/core/server';
```

Comment:
@chrisronline I'm super late to the party here, so forgive me for asking in the merged PR.
Is there a reason we're not adding core's metrics service `stats` too? In particular, core really want to access the `event_loop_delay` details for the histogram metrics.